OWNERS

```diff
@@ -13,6 +13,7 @@ reviewers:
   - raffo
   - seanmalloy
   - szuecs
+  - mloiseleur
```

Comment:
Should this not be in alphabetical order?

Reply:
There's no such requirement.

Reply:
Generally, when modifying a list that is already in alphabetical order I try to maintain that order.